src/utils/time.ts

```diff
@@ -222,7 +222,10 @@ export function isNightAtLocation(
     latitude: number,
     longitude: number,
     date: Date = new Date(),
-): TimeCheck {
+): {
```

Comment:
Could you please make `isNightAtLocation` to return boolean only?

Also, maybe if you try adding two alarms, one for the start (sunset) time, and another for end (sunrise time), you will not have to check for `nextCheck` every time.

In future it was planned to add an ability to switch between dark/light color scheme. Who knows, maybe we will consider adding more automation options, time intervals in future.

Reply:
> Could you please make isNightAtLocation to return boolean only?

The logic deciding whether it is no=ight or day right now and time of the next check both need results of `getSunsetSunriseUTCTime()`, so I can move call to `getSunsetSunriseUTCTime()` out of `isNightAtLocation` and then pass result into `isNightAtLocation` and funcion determining next time.

> Also, maybe if you try adding two alarms, one for the start (sunset) time, and another for end (sunrise time), you will not have to check for nextCheck every time.

These would still have to be one-time allarms (as opposed to periodic ones) because sunset/sunrise drift slightly during the year.

Reply:
The problem is, the answer for the question `is night at location` should be only `yes` or `no` (`true` or `false`). By looking at `rightNow` property name someone can guess its type is Date.

If it's not possible to just return boolean, then rename the function some way, like `getLocationSunInfo` or similar, that will return `{isNight: boolean; nextSunsetOrSunriseTime: Date}`.